### PR TITLE
Add remind command

### LINKED ISSUE #101: Add remind command

### PR DESCRIPTION
**Pull request by jannaleong**
Fixes #101

### CODE REVIEW

File: src/main/java/seedu/address/logic/messages/NoteMessages.java [hunk 1]
```diff
@@ -1,9 +1,35 @@
...
+     * Formats the {@code person} for display to the user.
+     */
+    public static String format(Person person) {
+        final StringBuilder builder = new StringBuilder();
+        if (person instanceof Staff) {
+            builder.append("Pooch Staff ");
+        } else if (person instanceof Supplier) {
+            builder.append("Supplier ");
+        } else if (person instanceof Maintainer) {
+            builder.append("Maintenance Crew ");
```
Comment (yleeyilin) [inline src/main/java/seedu/address/logic/messages/NoteMessages.java:28]: Use maintainer here, we are standardising to maintainer.

**Review by yleeyilin — COMMENTED** [review]
Minor issue, will approve after rectification. Otherwise, looks good!

File: src/main/java/seedu/address/logic/messages/NoteMessages.java (hunk 1 above)
Comment (jannaleong) [reply src/main/java/seedu/address/logic/messages/NoteMessages.java]: Noted!! i have made the changes.

**Review by yleeyilin — APPROVED** [review]
LGTM!

**Review by jamessinmaojun — APPROVED** [review]
lgtm

**Pull request merged**